COMPATIBILITY.md

```diff
@@ -1,13 +1,17 @@
-# Plugin Compatibility with Workflow
+# Workflow Compatibility Plugins
```

Comment:
Seems more confusing than the original. These are not plugins that offer compatibility with Workflow. They are plugins that existed independently, and here we are listing whether and how they are compatible with Workflow.